MekHQ/src/mekhq/campaign/personnel/death/AbstractDeath.java

```diff
@@ -252,11 +254,12 @@ private void initializeCausesFromFile(final File file) {
             if (!wn.hasChildNodes()) {
                 continue;
             }
+
             try {
                 final Gender gender = Gender.valueOf(wn.getNodeName());
                 getCauses().putIfAbsent(gender, new HashMap<>());
                 final NodeList nl2 = wn.getChildNodes();
-                for (int j = 0; j < nl.getLength(); j++) {
+                for (int j = 0; j < nl2.getLength(); j++) {
```

Comment:
Can we get better variable names other than `nl2` and `nl`? I know they mean `NodeList` bust still...

Reply:
It's a legacy naming setup, and one I don't want to fix currently. It is annoying, but standardization here is better for now.

Reply:
Ugh. Maybe one day.